internal/cmd/asyncapi/command_import.go

```diff
@@ -0,0 +1,426 @@
+	XMessageCompatibility string        `yaml:"x-messageCompatibility"`
+	Bindings              TopicBindings `yaml:"bindings"`
+}
+
+type TopicBindings struct {
+	Kafka kafkaBinding `yaml:"kafka"`
+}
+
+func newImportCommand(prerunner pcmd.PreRunner) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "import <input-file-name>",
+		Short: "Adds topics, schemas and tags from the yaml file.",
+		Args:  cobra.ExactArgs(1),
+		Example: examples.BuildExampleString(
+			examples.Example{
+				Text: "Import an AsyncAPI specification file to populate an existing cluster",
+				Code: "confluent asyncapi import spec.yaml",
+			},
+		),
+	}
+	c := &command{pcmd.NewAuthenticatedStateFlagCommand(cmd, prerunner)}
+	cmd.RunE = c.asyncapiImport
+	cmd.Flags().Bool("overwrite", false, "Overwrite existing topics and schemas.")
+	cmd.Flags().String("kafka-api-key", "", "Kafka cluster API key.")
+	cmd.Flags().String("schema-registry-api-key", "", "API key for Schema Registry.")
+	cmd.Flags().String("schema-registry-api-secret", "", "API secret for Schema Registry.")
+
+	return cmd
+}
+
+func getFlagsImport(cmd *cobra.Command) (*flagsImport, error) {
+	overwrite, err := cmd.Flags().GetBool("overwrite")
+	if err != nil {
+		return nil, err
+	}
+	kafkaApiKey, err := cmd.Flags().GetString("kafka-api-key")
+	if err != nil {
+		return nil, err
+	}
+	schemaRegistryApiKey, err := cmd.Flags().GetString("schema-registry-api-key")
+	if err != nil {
+		return nil, err
+	}
+	schemaRegistryApiSecret, err := cmd.Flags().GetString("schema-registry-api-secret")
+	if err != nil {
+		return nil, err
+	}
+
+	flags := &flagsImport{
+		overwrite:               overwrite,
+		kafkaApiKey:             kafkaApiKey,
+		schemaRegistryApiKey:    schemaRegistryApiKey,
+		schemaRegistryApiSecret: schemaRegistryApiSecret,
+	}
+	return flags, nil
+}
+
+func (c *command) asyncapiImport(cmd *cobra.Command, args []string) error {
+	spec, err := fileToSpec(args[0])
+	if err != nil {
+		return err
+	}
+	// Get flags
+	flagsImp, err := getFlagsImport(cmd)
+	if err != nil {
+		return err
+	}
+	// Getting Kafka Cluster & SR
+	flagsExport := flags{
+		kafkaApiKey:             flagsImp.kafkaApiKey,
+		schemaRegistryApiKey:    flagsImp.schemaRegistryApiKey,
+		schemaRegistryApiSecret: flagsImp.schemaRegistryApiSecret,
+	}
+	details, err := c.getAccountDetails(&flagsExport)
+	if err != nil {
+		return err
+	}
+	for topicName, topicDetails := range spec.Channels {
+		err = c.addChannelToCluster(details, spec, topicName, topicDetails.Bindings.Kafka, false, flagsImp.overwrite)
+		if err != nil {
+			if err.Error() == parseErrorMessage {
+				log.CliLogger.Info(err)
+			} else {
+				log.CliLogger.Warn(err)
+			}
+		}
+	}
+	output.Printf("AsyncAPI specification imported from \"%s\"\n", args[0])
+	return nil
+}
+
+func fileToSpec(fileName string) (*Spec, error) {
+	asyncSpec, err := os.ReadFile(fileName)
+	if err != nil {
+		return nil, err
+	}
+	spec := new(Spec)
+	err = yaml.Unmarshal(asyncSpec, &spec)
```

Comment:
You're taking a pointer of a pointer here...
```suggestion
	err = yaml.Unmarshal(asyncSpec, spec)
```